Dockerfile

```diff
@@ -189,18 +176,14 @@ RUN echo "${VERSION}" > /src/fmriprep/fmriprep/VERSION && \
     echo "include fmriprep/VERSION" >> /src/fmriprep/MANIFEST.in && \
     pip install --no-cache-dir "/src/fmriprep[all]"
 
-RUN install -m 0755 \
-    /src/fmriprep/scripts/generate_reference_mask.py \
-    /usr/local/bin/generate_reference_mask
```

Comment:
Are we making this available in the container in another way? This has in the past been useful for helping debug the reference mask generation when it goes badly.

Reply:
wasn't this ported over to niworkflows already? https://github.com/nipreps/niworkflows/blob/3120884336dd9c3b6211eeb538ed5f2d1085ac78/setup.cfg#L52-L53

Reply:
Oh, maybe. Have we tested that? It looks like it might not work, since there's no data sink.

https://github.com/nipreps/niworkflows/blob/master/niworkflows/cli/boldref.py

Reply:
the files should be available (albeit hidden) since we are setting the workflow base to the working directory. I opened up https://github.com/nipreps/niworkflows/issues/579 to address this concern.

In the meantime, I feel pretty comfortable removing this redundancy given that the tools are already in place should someone really need to use this under-the-radar helper script.

Reply:
Sure. I think the issue is going to be with the default of `remove_unnecessary_outputs=True` in the nipype config. But I agree that we can move this discussion over to niworkflows.